### ISSUE TITLE
Question about real training data

### ISSUE THREAD

**Issue by doughtmw** [jonashein/baseline_combination#1]
Hi Jonas,

Thank you for your awesome work on this project and for making the code and training data available! 

I had a question regarding the real train dataset - as a sanity check, I've been visually ensuring that the virtual 3D bounding boxes align well with the drill object. From your paper, I gathered that you used ICP for estimation of the ground truth drill pose. I would expect that this could introduce some error in the ground truth pose relative to the "actual" drill pose and wanted to ensure that the error that I'm experiencing is similar to what's expected. I've included a few sample images below with their respective bounding boxes drawn, do you think that the accuracy of ground truth placement looks reasonable in these examples? 

#### Good alignment
![10](https://user-images.githubusercontent.com/25186390/118701371-a34a1d00-b7e1-11eb-80c3-9e069fb7ad33.png)    

#### Poor alignment - looks like mostly rotational error introduced, more visible at instrument tip
![22](https://user-images.githubusercontent.com/25186390/118701407-ae9d4880-b7e1-11eb-96e3-b9c96c3814c6.png)  ![333](https://user-images.githubusercontent.com/25186390/118701801-23708280-b7e2-11eb-851e-57bf38f6d194.png) 

#### Sample from the virtual dataset
![25](https://user-images.githubusercontent.com/25186390/118702461-f2dd1880-b7e2-11eb-98c3-cea4171921d9.png)

Thanks for your help!

**Comment by jonashein** [comment]
Hi Mitchell,

yes, there are some errors in the ground truth for some of the real samples. The samples you provided look reasonable to me.

Besides the errors that could have been introduced by ICP (as you expected), another problem was the Kinects' unstable detection of the drill surface (e.g. due to reflections). This problem can be mitigated by using more cameras in the recording setup, however we only had two Kinects available at the time. As a result, parts of the drill were missing in the reconstructed point cloud for some frames, which also degraded the pose computed via ICP. We considered removing these inaccurate samples from the dataset, but in the end decided against this step as it would further reduce its size.

**Comment by doughtmw** [comment]
Awesome! Thanks for clearing that up, Jonas.
